packages/core/src/components/cv-tag/cv-tag.vue

```diff
@@ -37,6 +39,20 @@ const componentsTags = [
   'warm-gray',
 ];
 
+const tagTypes = [
```

Comment:
Why have we introduced this tagTypes list duplicating componentsTags which probably should be renamed tagKinds